AndrewTehan/hometask-5/4_2.sql

```diff
@@ -0,0 +1,7 @@
+select * from film
```

Comment:
films and languages are connected with 1-1 relationship, so `AND` doesn't suit there